var/spack/repos/builtin/packages/py-ipywidgets/package.py

```diff
@@ -37,4 +38,6 @@ class PyIpywidgets(PythonPackage):
     depends_on('py-widgetsnbextension@3.4.0:3.4', type=('build', 'run'),
                when='@7.4.2')
     depends_on('py-widgetsnbextension@3.5.0:3.5', type=('build', 'run'),
-               when='@7.5.1:')
+               when='@7.5.1:7.6.5')
+    depends_on('py-widgetsnbextension@3.6.0', type=('build', 'run'),
```

Comment:
```suggestion
    depends_on('py-widgetsnbextension@3.6', type=('build', 'run'),
```
`3.6` and `3.6.0:3.6` are both equivalent to `~=3.6.0` used in the setup.py. Exactly `3.6.0` is too strict.